metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_query.py

```diff
@@ -0,0 +1,59 @@
+class FivetranLogQuery:
+    @staticmethod
+    def use_schema(db_name: str, schema_name: str) -> str:
+        return f'use schema "{db_name}"."{schema_name}"'
+
+    @staticmethod
+    def get_connectors_query() -> str:
+        return """
+        SELECT connector_id as "CONNECTOR_ID",
```

Comment:
do these queries work multiple underlying databases?

Reply:
Haven't checked that but syntactically it should work as most of the destinations use SQL query language.
If there is some syntax which is not common, we can modify logic here in future to get different query for different destination.